src/main.rs

```diff
@@ -42,7 +42,13 @@ fn main() {
     let sys = actix::System::new("transpo-rt");
     let params = Params::from_args();
     let bind = format!("{}:{}", &params.bind, &params.port);
-    let context = transpo_rt::server::make_context(&params.gtfs, &params.url);
+
+    let today = chrono::Local::today(); //TODO use the timezone's dataset ?
+    let period = transpo_rt::context::Period {
+        begin: today.naive_local(),
+        end: today.succ().naive_local(),
```

Comment:
Comme l’inégalité est stricte, on est que sur 24h, non ? donc si le serveur tourne 20h, ça devient limite comme utilisabilité, non ?